Assets/Scripts/Bullets/BulletController.cs

```diff
@@ -1,24 +1,39 @@
-﻿using UnityEngine;
+﻿using System.Collections;
+using UnityEngine;
 
 public class BulletController : MonoBehaviour
 {
+    [SerializeField]
+    private ParticleSystem explosionParticle;
+    [SerializeField]
+    private AudioSource bulletExplosionAudio;
+
+
+
     [SerializeField]
     private float fireSpeed = 20f;
 
 
+
     public void Fire(Transform fireTransform, GameObject tank)
```

Comment:
Pass the tank transform as a parameter or just give the direction that the bullet needs to go in